version-mapping.json

```diff
@@ -27,8 +27,12 @@
     "index_image": "brew.registry.redhat.io/rh-osbs/iib:446618",
     "bundle_version": "v4.12.2-7"
   },
-  "2.6": {
-    "index_image": "brew.registry.redhat.io/rh-osbs/iib:351585",
-    "bundle_version": "v2.6.11-63"
+  "4.13": {
+    "index_image": "brew.registry.redhat.io/rh-osbs/iib:361684",
+    "bundle_version": "v4.13.0.rhel9-780"
```

Comment:
maybe you want to put a more up-to-date version?
(e.g. `v4.13.0.rhel9-1689`)

Reply:
done